src/main/resources/static/api.yaml

```diff
@@ -1323,3 +1355,42 @@ components:
       type: array
       items:
         $ref: '#/components/schemas/ConsentArtefactRepresentation'
+    HINotification:
+      type: object
+      properties:
+        transactionId:
+          type: string
+        doneAt:
+          type: string
+          format: date-time
+        notifier:
+          type: object
+          properties:
+            type:
+              type: string
+              enum: [HIU, HIP]
+            id:
+              type: string
+              example: tmh
+        statusNotification:
+          type: object
+          properties:
+            sessionStatus:
+              type: string
+              enum: [TRANSFERRED, FAILED]
+            hipId:
+              type: string
+              example: max
+            statusResponses:
+              type: array
+              items:
+                type: object
+                properties:
+                  careContextReference:
+                    type: string
+                  hiStatus:
+                    type: string
+                    enum: [DELIVERED, OK, ERRORED]
+                    example: OK
+                  description:
+                    type: string
```

Comment:
I think since we don't know what consent-manager is going to with the detailed response, can we defer the decision to include this till we get better understanding about that?